src/Web/WebSPA/Program.cs

```diff
@@ -81,16 +24,12 @@
 }
 
 app.UseRouting();
-app.MapDefaultControllerRoute();
-app.MapControllers();
-app.MapHealthChecks("/liveness", new HealthCheckOptions
-{
-    Predicate = r => r.Name.Contains("self")
-});
-app.MapHealthChecks("/hc", new HealthCheckOptions()
+
+#pragma warning disable ASP0014 // Suggest using top level route registrations
+app.UseEndpoints(routes =>
 {
-    Predicate = _ => true,
-    ResponseWriter = UIResponseWriter.WriteHealthCheckUIResponse
+    // TODO: Change this route
+    routes.MapGet("/home/configuration", (IOptions<AppSettings> options) => options.Value);
```

Comment:
Changed from `IOptionsSnapshot<T>` to `IOptions<T>`. In practice I don't think they'll ever change at runtime, but if we are setting best practice, maybe we should allow for that.
```suggestion
    routes.MapGet("/home/configuration", (IOptionsSnapshot<AppSettings> options) => options.Value);
```

Reply:
`IOptionsSnapshot<T>` is inefficient in the common case (where options do not change). We could use `IOptionsMonitor<T>` instead, but these options will not change in practice, and the code clutter is not worthwhile for the approximately zero benefit it would provide.